.circleci/config.yml

```
@@ -580,6 +607,89 @@ jobs:
               docker push fundocker/joanie:latest
             fi
 
+  hub-admin:
+    docker:
+      - image: cimg/base:2023.10
+    working_directory: ~/joanie
+    steps:
+      # Checkout repository sources
+      - checkout:
+          path: ~/joanie
+      # Generate a version.json file describing app release
+      - <<: *generate-version-file
+      # Activate docker-in-docker
+      - setup_remote_docker:
+          version: 20.10.23
```

Also use default like above with the job `build-docker-admin` ?
```suggestion
          version: default
```

---

Definitely ! Good catch